extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -62,6 +62,12 @@ export class ProjectsController {
 			newProject = await Project.openProject(projectFile.fsPath);
 			this.projects.push(newProject);
 
+			// open any reference projects (don't need to worry about circular dependencies because those aren't allowed)
+			const referencedProjects = newProject.databaseReferences.filter(r => r.databaseReferenceType === DatabaseReferenceType.project);
+			for (const proj of referencedProjects) {
+				await this.openProject(vscode.Uri.file(path.join(newProject.projectFolderPath, proj.fsUri.fsPath)));
+			}
```

Comment:
Two questions
1. if one open proj fails does it contine or stop
2. proj.fsUri.fsPath - we can be sure that this is relative to main project folder?

Reply:
1. added try catch so that failing to load a dependency doesn't make it stop
2. If the reference is being added through the UI in SSDT and eventually in ADS, then we can assume the path is relative to the main project folder. Should we also be supporting if people are adding these references manually with absolute paths?